hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/fs/TestFcHdfsSetUMask.java

```diff
@@ -100,16 +97,6 @@ public static void ClusterShutdownAtEnd() throws Exception {
       cluster.shutdown();
     }
   }
-  
-  {
-    try {
-      GenericTestUtils.setLogLevel(FileSystem.LOG, Level.DEBUG);
-    }
-    catch(Exception e) {
-      System.out.println("Cannot change log level\n"
-          + StringUtils.stringifyException(e));
-    }
-  }
```

Comment:
Not good practice to modify this manually.  Should be using log4j properties files to set logging levels.

Reply:
its a test and they may have had some good reasons, just as some of the Azure tests do

Reply:
We need to fix it properly then. @belugabehr do you mind undoing this particular one in a separate JIRA or proposing the log4j solution?